json struct.py

```diff
@@ -144,4 +168,4 @@ def generate_cpp_file(output_filename, configuration):
     
     if len(configuration) > 0:
         generate_header(output_filename, configuration)
-        generate_cpp_file(output_filename, configuration)
```

Comment:
Again, it looks like the review is inverting reality. I assume you terminated the file with a new line. Very weird.

Reply:
Yes, I terminated the file with a newline, where there was none before. GitHub has a weird way of displaying such changes. GitLab does too, actually.